demos/super_resolution_demo/main.cpp

```diff
@@ -139,11 +165,16 @@ int main(int argc, char *argv[]) {
             auto lrInputInfoItem = inputInfo[lrInputBlobName];
             int w = static_cast<int>(lrInputInfoItem->getTensorDesc().getDims()[3]);
             int h = static_cast<int>(lrInputInfoItem->getTensorDesc().getDims()[2]);
+            int c = static_cast<int>(lrInputInfoItem->getTensorDesc().getDims()[1]);
 
             if (w != img.cols || h != img.rows) {
                 slog::warn << "Size of the image " << i << " is not equal to WxH = " << w << "x" << h << slog::endl;
                 continue;
             }
+            if (c != img.channels()) {
+              slog::warn << "Number of channels of the image " << i << " is not equal to " << c <<slog::endl;
```

Comment:
Should the image just be converted to gray scale/BGR depending on used network? Or is there a limitation for nets and images should not be converted?

Reply:
Yes, it's limitation for nets.

Reply:
Two spaces are missed for this indent.

Reply:
Fixed